modules/graderStake/s1_util.go

```diff
@@ -40,6 +42,14 @@ func ValidateS1(entryhash []byte, extids [][]byte, height int32, content []byte)
 	}
 	o := &spr.S1Content{V2Content: *o2}
 
+	if extids[0][0] == 6 {
```

Comment:
Is this still relevant if we're splitting out version 6 signature checking into s2_util?

Reply:
`ValidateS1()` is updated.